services/spar/src/Spar/Scim/User.hs

```diff
@@ -447,9 +447,8 @@ createValidScimUser tokeninfo@ScimTokenInfo {stiTeam} vsu@(ST.ValidScimUser veid
           -- up.  We should consider making setUserHandle part of createUser and
           -- making it transactional.  If the user redoes the POST A new standalone
           -- user will be created.}
-          do
-            BrigAccess.setHandle buid handl
-            BrigAccess.setRichInfo buid richInfo
+          BrigAccess.setHandle buid handl
+          BrigAccess.setRichInfo buid richInfo
```

Comment:
I disagree with this commit: the `do`s are here intentionally to make it clear the contained lines form a self-contained thing.  But I think I'm a minority with this, so fine, keep this it.

Reply:
I'm pretty sure I introduced the `do` a few commits back to avoid two calls to `wrapMonadClientSem`.

Reply:
Oops, sorry, reflex.  I've been trying to defend this pattern before.  Never mind, my bad.